xls2xml/xls2xml/utils.py

```diff
@@ -19,3 +23,135 @@ def header_to_xml_tag(header):
     # cannot contain spaces (same as above)
 
```

Comment:
i guess tag = re.sub('^xml', '_', tag, flags=re.IGNORECASE) this case should be last . if the header contain spaces in the beginning then this wont work.

Reply:
The leading spaces will be later replaced by underscores, not trimmed. For exmaple, the "  xml" will become "__xml" and hence it should work in this case

Reply:
sorry my bad  :(